teps/0079-tekton-catalog-support-tiers.md

```diff
@@ -100,14 +122,283 @@ requirements, and provenance attestations can be made to meet software supply ch
 
+## Proposal
+
+### Ownership and Maintenance
+
+As previously discussed in [TEP-0003][tep-0003-upstream], we propose creating 
+two support tiers, `Community` and `Official`, through separate Tekton Catalogs.
+
+A Catalog is a git repository that complies with the organization contract 
+defined in [TEP-0003: Tekton Catalog Organization][tep-0003-org]. 
+
+The Community Catalog makes it easy for contributors to share resources, 
+while the Official Catalog provides a corpus of high quality resources 
+that users can rely on.
+
+Both the Community and Official Catalogs will be:
+* Owned by the [Tekton Catalog Maintainers][catalog-owners].
+* Published in the [Tekton Hub][hub].
+* Share infrastructure, such as testing and scanning tooling. 
+
+A given resource starts in the Community Catalog. Then it may get 
+promoted to the Official Catalog if:
+* It meets the [requirements of the Official Catalog](#requirements-1).
+* The [Tekton Catalog Maintainers][catalog-owners] approve its promotion. 
+
+If a resource is promoted to the Official Catalog, it can be 
+[deprecated][deprecation] in the Community Catalog.
+We will provide tooling, through [Caitlin][caitlin], to make it easier 
+to migrate from the Community Catalog to the Official Catalog.
+
+#### Tekton Community Catalog
+
+To maintain the low barrier to making contributions, users can easily 
+submit resources to the Community Catalog.
+
+The current Tekton Catalog at `tektoncd/catalog` will be reused as the 
+Community Catalog.
+
+##### Requirements
+
+1. The resource must comply with the contract defined in
+   [TEP-0003: Tekton Catalog Organization][tep-0003-org].
+2. The resource must define an OWNER file in 
+   `/{resource-type}/{resource-name}/OWNERS`, that specifies 
+   at least one maintainer for that specific resource.
+
+Note that resources in the Community Catalog can meet some
+[requirements](#requirements-1) of resources in the Official Catalog.
+
+#### Tekton Official Catalog
+
+To provide a corpus of high quality resources that users can rely on, 
+we propose creating an Official Catalog with high maintenance, testing 
+and security standards. 
+
+The Official Catalog will be in a new repository named 
+`tektoncd/catalog-official`. 
+
+##### Requirements
+
+These are requirement that a resource must meet to be added to the 
+Official Catalog:
+
+1. The resource must comply with the contract defined in
+   [TEP-0003: Tekton Catalog Organization][tep-0003-org].
+2. The resource must not define an OWNER file, given that it is owned 
+   and maintained by Tekton Catalog Maintainers who are defined at the 
+   root of the Catalog. The ownership responsibilities are described
+   [below](#responsibilities).
+3. The resource must have automated testing using Tekton for dogfooding. 
+   If there are any failures, they must be resolved as soon as possible; 
+   the SLO is one week. The automated testing is discussed further 
```

Comment:
under what circumstance will there be an automated failure that needs to be patched? i think this implies that there will be some kind of periodic testing here (e.g. nightly - im guessing details will go in the dogfooding section about this)

also, whenever we introduce something like periodic testing we need to do something to make sure ppl actually see these results - whether that's automated notifications to slack and/or some kind of build captain rotation (maybe you're going to dig into this in the automated testing section)

Reply:
yes, planning to get into the testing and dogfooding in the next part of the proposal -- planning to have periodic testing and we were discussing surfacing the results in the hub (slack notifications is a great idea)